test/taskext/main.d

```diff
@@ -0,0 +1,79 @@
+/*******************************************************************************
+
+    Test for ocean.util.app.ext.TaskExt
+
+    Copyright:
+        Copyright (c) 2017 sociomantic labs GmbH.
+        All rights reserved.
```

Comment:
Shouldn't it be written in one line?
`Copyright: Copyright (c) 2017 sociomantic labs GmbH. All rights reserved`

Reply:
DDOC should render it identically.

Reply:
Fair enough